<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>linucom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/new/prefix2/src/logo.png</normaloff>:/new/prefix2/src/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QComboBox" name="portNameComboBox">
      <item>
       <property name="text">
        <string>ttyS0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyS1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyS2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB0</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1" rowspan="11" colspan="5">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="baudRateComboBox">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>19</number>
      </property>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>75</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>110</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>134</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>150</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>76800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="dataBitsComboBox">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="parityComboBox">
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奇</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>偶</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QComboBox" name="stopBitsComboBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="obocheckBox">
      <property name="text">
       <string>连续发送</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>间隔(毫秒)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QSpinBox" name="delayspinBox">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1" colspan="5">
     <widget class="QLineEdit" name="sendMsgLineEdit">
      <property name="maxLength">
       <number>10000</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QPushButton" name="sendmsgBtn">
      <property name="text">
       <string>发送</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="5">
     <widget class="QPushButton" name="clearUpBtn">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QCommandLinkButton" name="myWebsiteCommandLinkButton">
      <property name="text">
       <string>在这里推广您的产品</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="ccradioButton">
      <property name="toolTip">
       <string>以字符的方式发送和接收数据</string>
      </property>
      <property name="text">
       <string>字符-字符模式</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="chradioButton">
      <property name="toolTip">
       <string>以字符方式发送, 以十六进制方式显示</string>
      </property>
      <property name="text">
       <string>字符-HEX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="cdradioButton">
      <property name="toolTip">
       <string>以字符方式发送, 以十进制方式显示</string>
      </property>
      <property name="text">
       <string>字符-DEC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="dcradioButton">
      <property name="toolTip">
       <string>将发送的数值转换为字符显示</string>
      </property>
      <property name="text">
       <string>DEC-字符</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLCDNumber" name="recvbyteslcdNumber">
      <property name="toolTip">
       <string>已接收字节数</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作(&amp;C)</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoadfile"/>
    <addaction name="actionWriteToFile"/>
    <addaction name="separator"/>
    <addaction name="actionReadPortInterval"/>
    <addaction name="separator"/>
    <addaction name="actionClearBytes"/>
    <addaction name="actionCleanPort"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionReadPortInterval"/>
   <addaction name="actionLoadfile"/>
   <addaction name="actionWriteToFile"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionClearBytes"/>
   <addaction name="actionCleanPort"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/add.png</normaloff>:/new/prefix1/src/add.png</iconset>
   </property>
   <property name="text">
    <string>添加串口</string>
   </property>
   <property name="toolTip">
    <string>添加串口设备文件</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/open.png</normaloff>:/new/prefix1/src/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/stop.png</normaloff>:/new/prefix1/src/stop.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/reload.png</normaloff>:/new/prefix1/src/reload.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/mesage.png</normaloff>:/new/prefix1/src/mesage.png</iconset>
   </property>
   <property name="text">
    <string>关于linucom</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/save.png</normaloff>:/new/prefix1/src/save.png</iconset>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
   <property name="toolTip">
    <string>保存收发框内的数据</string>
   </property>
  </action>
  <action name="actionClearBytes">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/clearbytes.png</normaloff>:/new/prefix1/src/clearbytes.png</iconset>
   </property>
   <property name="text">
    <string>计数清零</string>
   </property>
   <property name="toolTip">
    <string>清零已收到的字节数</string>
   </property>
  </action>
  <action name="actionReadPortInterval">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/time.png</normaloff>:/new/prefix1/src/time.png</iconset>
   </property>
   <property name="text">
    <string>读取间隔</string>
   </property>
   <property name="toolTip">
    <string>设置读取串口数据的时间间隔</string>
   </property>
  </action>
  <action name="actionLoadfile">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/src/loadfile.png</normaloff>:/new/prefix1/src/loadfile.png</iconset>
   </property>
   <property name="text">
    <string>读取文件</string>
   </property>
   <property name="toolTip">
    <string>载入外部文件作为发送内容</string>
   </property>
  </action>
  <action name="actionCleanPort">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix2/src/cleanport.png</normaloff>:/new/prefix2/src/cleanport.png</iconset>
   </property>
   <property name="text">
    <string>清空串口</string>
   </property>
   <property name="toolTip">
    <string>清空串口中I/O数据</string>
   </property>
  </action>
  <action name="actionWriteToFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix2/src/write2file.png</normaloff>:/new/prefix2/src/write2file.png</iconset>
   </property>
   <property name="text">
    <string>写入文件</string>
   </property>
   <property name="toolTip">
    <string>将读取数据写入到文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
